src/components/atoms/BuildingImg1.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<img v-bind:src="img1.src" v-bind:alt="img1.alt" class="building__img1">
+	<img v-bind:src="require(`../../assets/images/${img1.src}`)" v-bind:alt="img1.alt" class="building__img1">
```

Comment:
why not directly pass src as string?

Reply:
why use v-bind? use shorthand `:`

Reply:
It doesn't work if I pass it as a string